SAPHana/man/SAPHanaSR.py.7

```diff
@@ -56,21 +56,39 @@ Usage, syntax or execution errors.
 .\"
 .SH EXAMPLES
 .PP
-* Simple example for permission in /etc/sudoers. HANA 1.0 Scale-Out, SID is SLE.
+* Example for specific permission in /etc/sudoers.d/SAPHanaSR .
+.br
+HANA 1.0 Scale-Out, SID is SLE. The attribute contains the lowercase-SID.
```

Comment:
If HANA 1.0 means the HANA version - should we still name that in man page updates? HANA 1.0 is out of SAPs maintenance for a longer period of time.